lib/DDG/Spice/Pubmed/Search.pm

```diff
@@ -0,0 +1,21 @@
+package DDG::Spice::Pubmed::Search;
```

Comment:
We can actually ditch the `::Search` here now that we don't need the other two packages anymore